config/environments/production.rb

```diff
@@ -24,6 +24,8 @@
   # Apache or NGINX already handles this.
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
+  config.serve_static_assets = true
```

Comment:
So how your local environment relates to production? Use it locally, but do not commit it in the repo